sdk/eventhub/event-hubs/test/eventHubConsumerClient.spec.ts

```diff
@@ -225,6 +225,44 @@ describe("EventHubConsumerClient", () => {
       await producerClient.close();
     });
 
+    describe("#close()", function(): void {
+      it("stops any actively running subscriptions", async function(): Promise<void> {
+        const subscriptions: Subscription[] = [];
+        const client = new EventHubConsumerClient(
+          EventHubConsumerClient.defaultConsumerGroupName,
+          service.connectionString,
+          service.path
+        );
+
+        // Spin up multiple subscriptions.
+        for (const partitionId of partitionIds) {
+          subscriptions.push(
+            client.subscribe(partitionId, {
+              async processError() {
+                /* no-op for test */
+              },
+              async processEvents() {
+                /* no-op for test */
+              }
+            })
+          );
+        }
+
+        // Assert that the subscriptions are all running.
+        for (const subscription of subscriptions) {
```

Comment:
No delay is needed here? Do you want to know if they at least passed through processInitialize?

Reply:
Nope, it is not needed. This test will fail as is without the above change, just need to make sure it's reported as running.